webp-imageio/src/main/java/com/luciad/imageio/webp/WebPDecoderOptions.java

```diff
@@ -15,14 +15,20 @@
  */
 package com.luciad.imageio.webp;
 
+import java.lang.ref.Cleaner;
+
 final class WebPDecoderOptions implements Runnable {
 
+  static private final Cleaner cleaner  = Cleaner.create();

```

Comment:
According to [Cleaner](https://docs.oracle.com/en/java/javase/24/docs/api/java.base/java/lang/ref/Cleaner.html)'s documentation ideally there is a single cleaner per a library (I'm assuming for performance reasons).
I'm not able to do any benchmarks, so basing on my lack of knowledge (😅) I'd say we should keep single cleaner instance if possible. Unless you can prove me wrong? 👀